yarn.lock

```diff
@@ -2,7 +2,6602 @@
 # yarn lockfile v1
```

Comment:
@roortest  I'm not sure why there is so much "dependencies" in this file but I think it's not correct. I removed this file on my local and used yarn to generated and I got just the govuk-frontend dependecy. Which I think is correct. Can you try on you local please?

Reply:
that is due to web packer installation  i did it before to resolve those assets issues 
removed and reinstalled yarn . every thing sorted now. thanks .